app/build.gradle

```diff
@@ -37,6 +37,7 @@ dependencies {
     compile 'com.github.bumptech.glide:glide:3.7.0'
     compile 'de.hdodenhof:circleimageview:2.1.0'
     compile 'org.apache.commons:commons-math3:3.6.1'
+    compile group: 'org.apache.commons', name: 'commons-lang3', version: '3.5'
```

Comment:
can't it be like compile 'org.apache.commons:commons-lang3:3.5'  ??
Also I see common math version to be 3.6.1, use that if its available.